src/common/snippets/src/op/scalar.cpp

```diff
@@ -18,13 +18,15 @@ void Scalar::validate_and_infer_types() {
     Constant::validate_and_infer_types();
     auto out_pshape = get_output_partial_shape(0);
     NODE_VALIDATION_CHECK(this, out_pshape.is_static(), "Scalar supports only static input shapes");
-    NODE_VALIDATION_CHECK(this, out_pshape.get_shape().empty() || ov::shape_size(out_pshape.get_shape()) == 1,
-                      "Scalar supports only one-element constants, got ", out_pshape.get_shape(),
-                      " shape");
+    NODE_VALIDATION_CHECK(this,
+                          out_pshape.get_shape().empty() || ov::shape_size(out_pshape.get_shape()) == 1,
```

Comment:
Is `get_shape()` used intentionally here?

Reply:
to_shape and get_shape do the same things for partial shape